src/components/Filters/components/TagsWrapper/tests/TagsWrapper.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {mountWithApp} from 'test-utilities';
+
+import {TagsWrapper} from '../TagsWrapper';
+import {VisuallyHidden} from '../../../../VisuallyHidden';
+
+const MockChild = () => <div />;
+
+describe('<TagsWrapper />', () => {
+  it('renders visually hidden component when shouldHide is true', () => {
+    const tagsWrapper = mountWithApp(
+      <TagsWrapper shouldHide>
+        <MockChild />
+      </TagsWrapper>,
+    );
+
+    expect(tagsWrapper).toContainReactComponentTimes(VisuallyHidden, 1);
```

Comment:
How strongly do you feel about needing the count in there? I don't think it adds much value (same for the other assertions in this file).

```suggestion
    expect(tagsWrapper).toContainReactComponent(VisuallyHidden);
```